src/BinaryParsers/ProgramDatabase/Pdb.cs

```diff
@@ -275,7 +275,18 @@ public bool IsSegmentWithIdWritable(uint addressSection)
         private HashSet<uint> GenerateExecutableSectionContribIds()
         {
             var result = new HashSet<uint>();
-            var enumSectionContribs = CreateDiaTable<IDiaEnumSectionContribs>();
+            IDiaEnumSectionContribs enumSectionContribs = null;
+
+            try
+            {
+                CreateDiaTable<IDiaEnumSectionContribs>();
+            }
+            catch (NotImplementedException) { }
```

Comment:
I'm concerned that you may be getting these NotImplementedExceptions because you tried to get SectionContribs from the wrong base symbol; DIA lets you create an `IDiaSymbol` to a whole bunch of different things, most of which support only some subset of `IDiaSymbol`'s interface.

When you said BinScope had this exception, did you observe that by debugging BinScope or are you inferring that based on its output?